src/pages/access-denied/access-denied.module.css

```diff
@@ -0,0 +1,6 @@
+h1.master-teachers-export-data-view {
```

Comment:
Why does this reference MT export data view? Same question for not-found module CSS

Reply:
whoops. this is why you don't copy files

Reply:
lol